packages/components/src/navigation/test/index.js

```diff
@@ -266,14 +261,14 @@ describe( 'Navigation', () => {
 			screen.getByRole( 'heading', { name: 'Home' } )
 		).toBeInTheDocument();
 		expect(
-			screen.getByRole( 'button', { name: 'Item 1' } ).parentElement
-		).toHaveClass( 'is-active' );
+			screen.getByRole( 'link', { current: 'page' } )
```

Comment:
Note: only "Item 1" and "Item 2" have `role="link"` since, in the example, their menu items are passed an `href` attribute.

The rest of the menu items are rendered as `button` as expected